myapp/app/controllers/application_controller.rb

```diff
@@ -29,4 +30,17 @@ def logged_in_user
 
     redirect_to login_path
   end
+
+  def maintenance_mode?
+    maintenance_mode = Constant.find_by(key: 'maintenance_mode')
+    return false if maintenance_mode.nil?
+    return true if maintenance_mode.value == 'true'
+
+    false
```

Comment:
↓これで上記3行とやっていることは同じだったりします。
```suggestion
    maintenance_mode&.value == 'true'
```
なぜ同じ結果になるのか考えてみてください。
修正は任意です。
修正する場合、1度しか触らないので `maintenance_mode` の代入も省いていいかもしれません。

Reply:
- rubyのメソッドの最後の式が戻り値となるという仕様
- ぼっち演算子「&.」で`maintenance_mode`がnilの場合はnilをreturnし、nilでない場合はvalueメソッドを実行して'true'かどうか判定する
- nilは偽判定される
- メソッドの戻り値が条件式の場合はtrue/falseが返される

これらを組み合わせると一行で書けるのですね。。勉強になります！
https://github.com/Fablic/training/pull/1352/commits/e5a5ee450058d25fb15429adb88e55d93a5eca84